### ISSUE TITLE
Coarse band shape for uncommon frequency resolutions.

### ISSUE THREAD

**Issue by d3v-null** [RadioAstronomySoftwareGroup/pyuvdata#1390]
Hi there,
When reading raw fits files with 1kHz Frequency resolution, and without `remove_coarse_band=False` I get this error.

```
ValueError: mwax passband shapes are only available for 40 kHz and 80 kHzfine channel widths. To request a passband for a different fine channel width create an issue on the pyuvdata repository. To run without a passband, resubmit with correct_coarse_band=False
```

This message appears to be out of date, as the `UVData.read()` argument has been renamed to `remove_coarse_band`. 

If anyone wants to implement the coarse band shape for uncommon frequency resolutions, here's some info about how it was done in Birli:

All currently available [correlator frequency resolution modes](https://mwatelescope.atlassian.net/wiki/spaces/MP/pages/24973003/MWAX+Modes) are a multiple of 200Hz. [The Birli implementation](https://github.com/MWATelescope/Birli/blob/f5976a4f00a168e7fcb471a44d2c997b2ffdec0f/src/corrections.rs#L459) of the PFB passband shape correction "scrunches" the 200Hz correction provided on [this page](https://mwatelescope.atlassian.net/wiki/spaces/MP/pages/24972979/RRI+Receiver+PFB+Filter) to the frequency resolution. 

Thanks

**Comment by d3v-null** [comment]
Just an update on this, not sure if pyuvdata takes this into account, but I recently learned that the pfb passband shape that works best for MWAX data does not perform as well as the cotter pfb shape (aka levine gains) for legacy data. This is due to subtle differences in the fine channelization in between mwax and legacy correlator. 

if pyuvdata is applying a different pfb shape for mwax observations that takes into account the fine channelization, then you can safely ignore this. otherwise, there are more details are in here: https://github.com/MWATelescope/Birli/issues/152

Thanks

**Comment by bhazelton** [comment]
@d3v-null Yes, we apply a different PFB correction for legacy and for MWAX data. @PyxieLouStar spent a lot of time with the documentation and with the ops team to work this out. She only implemented the 40 & 80 kHz corrections for MWAX but I think we could get an implementation for finer frequency resolution pretty straightforwardly. I've been a little consumed with trying to get version 3.0 done but I can try to take a look on this front once we get that out.

Or if you'd like it sooner and are interested, I can try to find the information you'd need to implement it and make a PR to get it into pyuvdata.
